src/main/java/com/jhuguet/sb_taskv1/app/controller/UserController.java

```diff
@@ -86,8 +92,11 @@ public User get(@PathVariable int id) throws IdNotFound {
      * @throws OrderNotRelated Exception thrown when given order ID is not associated to user
      */
     @GetMapping("/{userId}/orders/{orderId}")
-    public Order getOrder(@PathVariable int userId, @PathVariable int orderId) throws IdNotFound, OrderNotRelated {
+    public Order getOrder(@PathVariable int userId,
```

Comment:
Why do you pass userId as a param?

Reply:
I pass it as a parameter because I'm validating the id passed through the URL when compared to the one contained in the jwt

Reply:
I think we negotiated to use approach, when userId isn't passed , but is taken from token.